sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c

```diff
@@ -862,14 +862,14 @@ static void _check_for_ecn(gnrc_pktsnip_t *frame)
 static bool _send_frame(gnrc_pktsnip_t *frame, gnrc_sixlowpan_frag_fb_t *fbuf,
                         void *ctx, unsigned page)
 {
-    uint32_t now;
-    uint32_t if_gap = gnrc_sixlowpan_frag_sfr_congure_snd_inter_frame_gap(fbuf);
+    uint32_t now_msec;
+    uint32_t if_gap_msec = gnrc_sixlowpan_frag_sfr_congure_snd_inter_frame_gap(fbuf) / US_PER_MS;
```

Comment:
IIRC the inter frame gap is currently configured to be in the sub-millisecond range. How is this handled here?